cpp/cmake/thirdparty/get_cufile.cmake

```diff
@@ -0,0 +1,31 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+# This function finds nvcomp and sets any additional necessary environment variables.
+function(find_and_configure_cufile)
+
+  list(APPEND CMAKE_MODULE_PATH ${CUDF_SOURCE_DIR}/cmake/Modules)
+  rapids_find_package(cuFile QUIET)
+
+  if(cuFile_FOUND AND NOT BUILD_SHARED_LIBS)
+    rapids_export_find_package_file(
```

Comment:
I tried building the latest code in the Java Docker image but it fails to configure libcudf with:
```
CMake Error at cmake/thirdparty/get_cufile.cmake:22 (rapids_export_find_package_file):
  Unknown CMake command "rapids_export_find_package_file".
Call Stack (most recent call first):
  cmake/thirdparty/get_cufile.cmake:31 (find_and_configure_cufile)
  CMakeLists.txt:157 (include)
```